daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -490,8 +503,22 @@ codegenTests codegenDir = testGroup "daml codegen"
                     callCommandQuiet $ unwords ["daml new", projectDir, "skeleton"]
                     withCurrentDirectory projectDir $ do
                         callCommandQuiet "daml build"
-                        let darFile = projectDir</> ".daml/dist/proj-" ++ lang ++ "-0.0.1.dar"
-                            outDir  = projectDir</> "generated" </> lang
+                        let darFile = projectDir </> ".daml/dist/proj-" ++ lang ++ "-0.0.1.dar"
+                            outDir  = projectDir </> "generated" </> lang
+                        when (lang == "ts") $ do
```

Comment:
Can you please leave _one_ comment that explains what we're doing here, namely making `@daml-types-0.0.9-SDKVERSION` available to yarn, instead of having multiple comments spread out below.

Other than that, very nice solution!

Reply:
Ok